### ISSUE TITLE
Amphetype asks for untypable input

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gordonfierce/amphetype#13]
```
I use a wide variety of e-book texts, and the formatting conventions differ
greatly over them all. The most annoying single thing about this is that
often a extract will contain untypable input.

An example: My keyboard only has '-'; it doesn't have ‒, –, —, or ― (or,
for that matter, many of the characters listed in
https://secure.wikimedia.org/wikipedia/en/wiki/Hyphen or
https://secure.wikimedia.org/wikipedia/en/wiki/Dash )

So if Amphetype gives me a text selection containing any of those, I might
type satisfactorily up to the offending character and be stopped.

It'd be best for Amphetype to do something like translate everything into
ASCII which I can actually type.
```

Original issue reported on code.google.com by `gwe...@gmail.com` on 13 Feb 2009 at 5:12
